myintegrations/gitea-repositories/user1/repo1.git/hooks/post-receive.d/gitea

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+"/home/ethantkoenig/.gopath/src/code.gitea.io/gitea/gitea" hook --config='integrations/app.ini' post-receive
```

Comment:
Hmm, I didn't realize there were hard-coded absolute paths in here. If we pull from somewhere else, we'll still have to edit all of these scripts to have the correct path (which is doable, but annoying).